internal/util/debug/debug.go

```diff
@@ -56,11 +56,32 @@ func RunHandler(ctx context.Context, addr string, r prometheus.Registerer, l *za
 	}
 	must.NoError(statsviz.Register(http.DefaultServeMux, opts...))
 
+	http.HandleFunc("/debug/started", func(r http.ResponseWriter, _ *http.Request) {
```

Comment:
We want to know how those handlers are called. We should instrument them with promhttp